components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -37,18 +34,14 @@ class ToolbarPresenter(
 
     private var scope: CoroutineScope? = null
 
-    // This maps web extension id to [WebExtensionToolbarAction]
-    @VisibleForTesting
-    internal val webExtensionBrowserActions = HashMap<String, WebExtensionToolbarAction>()
-
     /**
      * Start presenter: Display data in toolbar.
      */
     fun start() {
         renderer.start()
 
         scope = store.flowScoped { flow ->
-            flow.ifAnyChanged { arrayOf(it.findCustomTabOrSelectedTab(customTabId), it.extensions) }
+            flow.ifAnyChanged { arrayOf(it.findCustomTabOrSelectedTab(customTabId)) }
```

Comment:
This is just observing one change now so it should be: 
`flow.ifChanged { it.findCustomTabOrSelectedTab(customTabId) }`